lib/linguist/heuristics.yml

```diff
@@ -513,6 +513,14 @@ disambiguations:
   rules:
   - language: SubRip Text
     pattern: '^(\d{2}:\d{2}:\d{2},\d{3})\s*(-->)\s*(\d{2}:\d{2}:\d{2},\d{3})$'
+- extensions: ['.st']
+  rules:
+  - language: StringTemplate
+    pattern:
+    - '</\w+>'
+    - '\$\w[($]'
+  - language: Smalltalk
+    pattern: '\A[\[{("\w\d#]|\w+\s*:=\s*\w+|class >> \w+|^\w+\s+\w+:|^Class\s*{|if(?:True|False):\s*\['
```

Comment:
@Alhadis mind casting your 👀 over this? 🙇

Reply:
Forgot a plus after line 521 \w. Also I assume having a list means "both must match".

Reply:
@lildude Done. Sorry for the unresponsiveness... I haven't been feeling well lately. 😞

Reply:
@lildude Done. Sorry for the unresponsiveness... I haven't been feeling well lately. 😞

Reply:
@lildude Done. Sorry for the unresponsiveness... I haven't been feeling well lately. 😞